### PR TITLE
TST: check feerate in onchain send fee selector

### PR DESCRIPTION
**Pull request by Overtorment**

### CODE REVIEW

File: tests/e2e/bluewallet.spec.js [hunk 1]
```diff
@@ -360,6 +360,14 @@ describe('BlueWallet UI Tests', () => {
     await element(by.id('SendButton')).tap();
     await element(by.id('AddressInput')).replaceText('bc1q063ctu6jhe5k4v8ka99qac8rcm2tzjjnuktyrl');
     await element(by.id('BitcoinAmountInput')).typeText('0.0001\n');
+
+    // setting fee rate:
+    const feeRate = 2;
+    await element(by.id('chooseFee')).tap();
+    await element(by.id('feeCustom')).tap();
```
Comment (limpbrains) [inline tests/e2e/bluewallet.spec.js:367]: No such elements, did you forget to add some files to commit ?

**Review by limpbrains — CHANGES_REQUESTED** [review]

File: tests/e2e/bluewallet.spec.js (hunk 1 above)
Comment (Overtorment) [reply tests/e2e/bluewallet.spec.js:367]: oops

**Comment by limpbrains** [comment]
this is cool

**Review by limpbrains — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/8gay58bwt4t7jwfkkv5nbc4p14

[download apk](http://filestorage.bluewallet.io:1488/1600181913.apk)

**Pull request merged**